build.gradle

```
@@ -1,5 +1,5 @@
 plugins {
-	id 'fabric-loom' version '0.6-SNAPSHOT'
+	id 'fabric-loom' version '0.7-SNAPSHOT'
```

```suggestion
	id 'fabric-loom' version '0.6-SNAPSHOT'
```
As far as I know, Loom 0.7 is in development and still is unstable, it should not be used